docs/src/test/java/docs/http/javadsl/HttpServerActorInteractionExample.java

```diff
@@ -105,13 +106,20 @@ private Route createRoute() {
 
   //#actor-interaction
   static class Auction extends UntypedActor {
+    List<Bid> bids = new ArrayList<>();
+
     static Props props() {
       return Props.create(Auction.class);
     }
 
     @Override
     public void onReceive(Object message) throws Throwable {
```

Comment:
Done, it is cleaner now. But I had to move `Auction` actor outside the `HttpServerActorInteractionExample` because of:
> [error] (run-main-0) java.lang.IllegalArgumentException: no matching constructor found on class docs.http.javadsl.HttpServerActorInteractionExample$Auction for arguments []
java.lang.IllegalArgumentException: no matching constructor found on class docs.http.javadsl.HttpServerActorInteractionExample$Auction for arguments []